platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/command/ProfileInfoCommand.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.admin.application.service.command;
+
+import static org.boon.Boon.toPrettyJson;
+
+import org.apache.karaf.features.Feature;
+import org.apache.karaf.shell.api.action.Argument;
+import org.apache.karaf.shell.api.action.Command;
+import org.apache.karaf.shell.api.action.lifecycle.Service;
+import org.codice.ddf.admin.application.service.ApplicationService;
+import org.codice.ddf.admin.application.service.ApplicationServiceException;
+import org.fusesource.jansi.Ansi;
+
+import java.util.List;
+import java.util.Map;
+
+@Command(scope = "admin", name = "profile-info", description = "Lists available install profiles")
+@Service
+public class ProfileInfoCommand extends AbstractProfileCommand {
+    @Argument(index = 0, name = "profileName", description = "Name of install profile to use", required = true, multiValued = false)
+    String profileName;
+
+    @Override
+    void doExecute(ApplicationService applicationService) throws ApplicationServiceException {
+        if (!profileExists(applicationService.getInstallationProfiles(), profileName)
+                && showDevProfiles && devProfileExists(profileName)) {
+            Map devProfile = getDevProfile(profileName);
+            devProfile.forEach((k, v) -> {
+                printSectionHeading(k + ":");
+                List<String> vals = (List<String>) v;
+                vals.forEach(console::println);
+            });
+        } else {
+            Feature profile = getProfile(applicationService, profileName);
+            printSectionHeading("Apps:");
```

Comment:
If anything we should probably have the devProfiles use the "apps" heading. unfortunately the dev profile sections equate to actions instead of just the type of thing being acted on. I think I'm going to leave this the way it is since anything in the regular profiles are technically the "real" headings